packages/create-cloudflare/src/templates.ts

```diff
@@ -203,10 +205,27 @@ export const selectTemplate = async (args: Partial<C3Args>) => {
 		question: "What would you like to start with?",
 		label: "category",
 		options: [
-			{ label: "Hello World example", value: "hello-world" },
-			{ label: "Framework Starter", value: "web-framework" },
-			{ label: "Demo application", value: "demo" },
-			{ label: "Template from a Github repo", value: "remote-template" },
+			{
+				label: "Hello World example",
+				value: "hello-world",
+				description:
+					"[Recommended for beginners] Build an empty starter to configure project structure and bare-bones",
+			},
+			{
+				label: "Framework Starter",
+				value: "web-framework",
+				description: "Create a website with a JavaScript framework",
```

Comment:
Perhaps something like `Create an application with a full-stack web framework` might be better